explorer_frontend/src/features/healthcheck/init.ts

```diff
@@ -0,0 +1,30 @@
+import { sample } from "effector";
+import { interval } from "patronum";
+import { $endpoint } from "../account-connector/model";
+import { loadedPlaygroundPage } from "../code/model";
+import { $isPageVisible, $rpcIsHealthy, checkRpcHealthFx, pageVisibilityChanged } from "./model";
+
+$isPageVisible.on(pageVisibilityChanged, (_, isVisible) => isVisible);
+document.addEventListener("visibilitychange", () => {
+  pageVisibilityChanged(document.visibilityState === "visible");
+});
+
+const { tick } = interval({
+  timeout: 1000 * 10,
+  start: loadedPlaygroundPage,
+  leading: true,
+});
+
+sample({
+  clock: tick,
+  target: checkRpcHealthFx,
+  source: $endpoint,
+  filter: $isPageVisible,
```

Comment:
also would be good to have isPageVisible and that is actually playground or related page. you could check this with $opened on route instance

Reply:
Done